packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -135,15 +135,15 @@ sealed class _DebugSemanticsRoleChecks {
     SemanticsRole.status => _noLiveRegion,
     SemanticsRole.list => _noCheckRequired,
     SemanticsRole.listItem => _semanticsListItem,
+    SemanticsRole.loadingSpinner => _noCheckRequired,
+    SemanticsRole.progressBar => _noCheckRequired,
```

Comment:
this should check if it has value, minValue, maxvalue. and ideally value should be within min and max

Reply:
Sure, let me push the change.